joomla/joomla-cms
# [4.0] Style legend the same as label

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/23888#issuecomment-464450325 .

### Summary of Changes
Style legend the same as label

### Testing Instructions
Unable to test with patchtester. Apply patch. Compile SCSS. Using dev tools replace field label with a legend. Ensure styling doesn't change.


### Expected result



### Actual result



### Documentation Changes Required
